docs/examples/benchmark-snakegame/run-benchmark.sh

```
@@ -1,40 +1,40 @@
-        if [ $? -ne 0 ]; then
-            time="error"
-            exit 1
-        fi
-
-        if [ "$lang" != "swift" ]; then
-            printf "%0.2f tps," $time
-        else
-            printf "%0.2f tps\n" $time
-        fi
-    done
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)